tracing.go

```diff
@@ -26,6 +26,11 @@ import (
 	opentracing "github.com/opentracing/opentracing-go"
 )
 
+const (
+	RpcYarpcStatusCode = "rpc.yarpc.status_code"
```

Comment:
That's the root namespace, let's call it `TracingTagStatusCode` for example. And let's add proper comment for a new const.